web/src/cypress/integration/favorites.ts

```diff
@@ -0,0 +1,107 @@
+import { testScreen } from '../support'
+import { Chance } from 'chance'
+const c = new Chance()
+
+testScreen('Favorites', testFavorites)
+
+function testFavorites(screen: ScreenFormat) {
+  describe('Rotation Favorites', () => {
+    let rot: Rotation
+    beforeEach(() => {
+      cy.createRotation()
+        .then(r => {
+          rot = r
+        })
+        .visit('/rotations')
+    })
+
+    it('should have favored rotations move to first on rotation list', () => {
+      cy.pageSearch(rot.name)
+      cy.get('#app')
+        .contains(rot.name)
+        .click()
+      cy.get('button[aria-label="Set as a Favorite Rotation"]')
+        .click()
+        .visit('/rotations')
+      cy.get('#app').contains(rot.name)
+    })
+
+    it('should have favorites first in RotationSelect on escaltion policy steps', () => {
+      cy.pageSearch(rot.name)
+      cy.get('#app')
+        .contains(rot.name)
+        .click()
+      cy.get('button[aria-label="Set as a Favorite Rotation"]')
+        .click()
+        .visit('/escalation-policies')
+
+      cy.pageFab()
+
+      cy.get('div[role=dialog]').as('dialog')
+      cy.get('@dialog').should('contain', 'Create Escalation Policy')
```

Comment:
we can utilize some cypress support functions we have created in the past to create an escalation policiy by directly querying Graphql, which should speed up this test's runtime. Before the test runs you can do something along the lines of:

```
let ep = null
cy.createEP().then(e => {
  ep = e
  cy.visit(`/escalation-policies/${ep.id}`)
  cy.reload()
})

// open dialog and check dropdown
```